### DISCUSSION TITLE
Expanding potential

### DISCUSSION THREAD

**Discussion by vssasks** [michaelwillis/dragonfly-reverb#149] [Ideas]
It's necessary to implement dual mono modes in all reverbs like in many existing reverbs like parallel mode in reverberate for example. Otherwise it always spoils phase between channels. And 2: which of them correspond to which on japanese site?
3. I don't use jackd and not going to. Can i compile plugs without jackd library installed?

**Comment by vssasks** [comment]
Anyway, no need. Gunshot convolution reverb and airwindows reverbs already do dualmono processing...

**Comment by michaelwillis** [comment]
1. Interaction between the two stereo channels is fundamentally baked into the algorithms at hand. To disentangle the two would be to create entirely new reverb algorithms.
2. Dragonfly Hall is based on Hibiki, Dragonfly Room is based on ProG, and Dragonfly Plate uses both NVerb (simple and nested modes) and STRev (tank mode). Dragonfly Early Reflections uses the ER algorithm found in both Hibiki and ProG.
3. You can modify the four `makefile`s found in the subdirectories under `plugins`. Remove `jack` from the following line in each of them: `TARGETS = jack lv2_sep vst2 vst3 clap`

**Reply by vssasks** [reply]
Ok, kind of another design vs reverb assistant where you can set comb and allpass filters to same settings and it turns into dual mono thing. I thought you could set same parms for both chls too. I just hate holding 2 stereo copies for sending via their left channel. Too consuming

**Reply by vssasks** [reply]
Also i wish you modify code to support buffer 32. As i remember he mentioned 64 as smalest buffer. I noow test gunshot reverb. I hope it can stand vs japanese' guy's ir reverb in terms of quality?

**Comment by michaelwillis** [comment]
These reverbs should work with any buffer size. Who mentioned 64 as the smallest supported size?

**Reply by vssasks** [reply]
I guess the author of easy reverb who made ir reverb not a long time ago. However i measuded and mostt reverbs from japanese guy had latency as i recall. By the way all algorithmic reverbs have metallic sound when 100%. I have not found a single one without that metallicity, even vss3 which was praised by many and indeed sounds good. So now I hunt for for linux compiled ir gpu dual mono reverb. Unfortunately no one has yet made one. As i recall japanese guy spoke about 1024 - 16000 buffers for freeverb3 ir reverb on his site for old pentiums in 2003. However part of his reverbs or all had latencies for which reason i had to reject them when choosing plugins.

**Reply by vssasks** [reply]
Also interested in for linux dynamic conv reverbs like that from electri-q's author not available anywhere and monopolized by AA as for technology. It could be kind of answer to quantec hardware.
